extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -163,5 +160,27 @@ amp-carousel .amp-carousel-button.amp-disabled {
   order: 100000000 !important;
   margin-right: -1px !important;
 }
-
 /* End SlideScroll */
+
+
+
+
+
+/* Begin ScrollableCarousel */
+.-amp-scrollable-carousel-container .amp-carousel-slide {
```

Comment:
it already exists :)
https://github.com/ampproject/amphtml/blob/a8acbd2dac5f8d2835c934e3f7adccd62902941a/extensions/amp-carousel/0.1/test/test-scrollable-carousel.js#L68